webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsAdminService.scala

```diff
@@ -0,0 +1,47 @@
+package org.knora.webapi.responders.admin
+import zio.Task
+import zio.ZIO
+import zio.ZLayer
+
+import org.knora.webapi.config.AppConfig
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectGetResponseADM
+import org.knora.webapi.messages.admin.responder.projectsmessages.ProjectIdentifierADM
+import org.knora.webapi.messages.admin.responder.usersmessages.UserADM
+import org.knora.webapi.responders.ActorDeps
+import org.knora.webapi.store.cache.settings.CacheServiceSettings
+
+trait ProjectsAdminService {
```

Comment:
If the "admin" in the name refers to the admin package (as opposed to v1 and v2), it should be called "ADM" and used as postfix like `ProjectsServiceADM`. At least this is how we did it so far, and I think it would be good to handle naming consistently.

Reply:
I found naming like `…AdminService` more speaking to me than `…ServiceADM`, after all it is a service and not a serviceADM (whatever this should mean). Maybe calling it simply `ProjectsService` is sufficient? I do find the `ADM` postfix quite disturbing.

Reply:
Whilst I agree that consistency is important I disagree with the notion of an exclusive either or choice here.

Aiming for consistency should not stop us from finding better names for new things without having to change the whole world immediately in small refactoring like this.

The `ADM` is fine for the old responders coming from the akka world and should this be bothering us we can still rename them later in a separate task/commit. In the zio world we might want to have a better/different naming.

Reply:
Let's try it out.